src/jasmineTestRunner/JasmineReporter.js

```diff
@@ -12,7 +12,17 @@ var diff = require('diff');
 var jasmine = require('../../vendor/jasmine/jasmine-1.3.0').jasmine;
 var Q = require('q');
 
-var colorize = colors.colorize;
+// Used in colorized mode.
+function _formatMsgColorized(msg, color) {
+  return colors.colorize(msg, color);
+}
+
+// Used in plain mode to output.
+function _formatMsgPlain(msg) {
+  return msg;
+}
+
+var _formatMsg;
```

Comment:
I don't like that this a module global variable. Can we make it either:

* an instance variable and then make the function using it a method
* Make it be passed as an argument to the function that's going to use it